src/mips/psyqo/examples/cdrom-demo/cdrom-demo.cpp

```diff
@@ -75,6 +76,11 @@ void CDRomDemo::createScene() {
             syscall_puts("Failure\n");
         }
     });
+
+
+    psyqo::Hardware::CDRom::CDRomCommandBuffer commandBuffer;
+    commandBuffer.set(0x50, 0xf2, 0xde, 0xad);
+    m_cdrom.testBlocking(gpu(), commandBuffer);
```

Comment:
I didn't realize I committed this change to the repo - is there a clean way to roll back this change or prevent it from getting merged in? Otherwise I can just revert the changes.

Reply:
Eh, no need to rewrite history for this. Just push a commit to remove it if you want it gone.